src/js/controller.js

```diff
@@ -470,13 +470,29 @@ export const MediaUIStates = {
       return getShowingSubtitleTracks(controller).map(({ kind, label, language }) => ({ kind, label, language }));
     },
     mediaEvents: ['loadstart'],
-    trackListEvents: ['addtrack', 'removetrack', 'change'],
+    textTracksEvents: ['addtrack', 'removetrack', 'change'],
+  },
+  MEDIA_RENDITION_LIST: {
+    get: function (controller) {
+      const { media } = controller;
+      return [...media?.videoRenditions ?? []];
+    },
+    mediaEvents: ['emptied'],
+    videoRenditionsEvents: ['addrendition', 'removerendition'],
+  },
+  MEDIA_RENDITION_SELECTED: {
+    get: function (controller) {
+      const { media } = controller;
+      return media?.videoRenditions?.[media.videoRenditions?.selectedIndex];
+    },
+    mediaEvents: ['emptied'],
```

Comment:
Is there no event we should be listening to for the first load? I worry a little about missing an add rendition event, if that's what's being relied on.

Reply:
I can see that. 
let's add `loadstart` as well